core/client/src/main/java/alluxio/hadoop/HdfsFileInputStream.java

```diff
@@ -47,6 +47,7 @@
 public class HdfsFileInputStream extends InputStream implements Seekable, PositionedReadable {
   private static final Logger LOG = LoggerFactory.getLogger(Constants.LOGGER_TYPE);
 
+  private final boolean mPacketInStreamEnabled;
```

Comment:
No particular reason. I was trying to be consistent with the the way we use site properties within this file such as USER_FILE_BUFFER_BYTES.